config/settings.yml

```diff
@@ -665,10 +665,10 @@ claims_api:
 
 connected_apps_api:
   connected_apps:
-    url: ~
+    url: https://sandbox-api.va.gov/internal/auth/v3/user/connected-apps
     api_key: ~
     revoke_url: https://sandbox-api.va.gov/internal/auth/v3/user/consent
-
+    
```

Comment:
We want to avoid unexpected whitespace here, can we revert to only include the url line adjustment?